x/concentrated-liquidity/lp_test.go

```diff
@@ -998,7 +998,7 @@ func (s *KeeperTestSuite) TestAddToPosition() {
 				amount1Minimum: sdk.NewInt(9999998916),
 				expectedError: types.InsufficientLiquidityCreatedError{
 					Actual:      sdk.NewInt(9999998816),
-					Minimum:     sdk.NewInt(9999998916),
+					Minimum:     sdk.NewInt(14999998915),
```

Comment:
Also, how are these values chosen? Please add a comment

Reply:
Added explanation and caculations per https://github.com/osmosis-labs/osmosis/pull/5267/commits/0c40801fa8c53145241ad7cf54c0711606081eea